infogami/infobase/client.py

```diff
@@ -203,6 +203,9 @@ def _get(self):
             self._o = self._creator()
         return self._o
 
+    def __repr__(self):
```

Comment:
This will make the test pass but are we going to have to call `list(lazy_object)` in other places in the future?  #88 was trying to enable both.

Reply:
Gotcha -- I'll add an implementation for `__iter__`, which seems to do the trick peacefully.

Reply:
Nice.